### DISCUSSION TITLE
New Beta Lists (for starred repos) Feature doesn't have a delete button

### DISCUSSION THREAD

**Discussion by kulom001** [community/community#43282] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

wanted to delete a list I made, saw it was in beta and I could report lack of button. Thanks for all the hard work btw, much appreciated! If I made a mistake (I do that a lot, please let me know). Just looking out for the platform!).

**Comment by radar-blip** [comment, marked as answer]
Once you click on a list, then choose the `Edit list` button, an option to `Delete list` should be there. Like below:

![deletelist](https://user-images.githubusercontent.com/39196767/210507710-bb06eb71-7cc7-487d-9b58-602aac678e9c.png)

**Comment by kulom001** [comment]
thanks

**Reply by kulom001** [reply]
how do i close discussion?

**Reply by kulom001** [reply]
got it

**Reply by radar-blip** [reply]
You're welcome. Glad to help!

I dont think you can close this discussion. It will remain here as it might be helpful to others.
